File: codeforces/10-805b-3-palindrome.go
package main

import "fmt"

func main() {
	var size int
	fmt.Scanf("%d\n", &size)

	aabb := "aabb"
	for i := 0; i < size; i++ {
		fmt.Printf("%c", aabb[i%4])
	}
	fmt.Printf("\n")
}

File: codeforces/10-805b-3-palindrome.cpp
#include <iostream>
#include <string>

int main() {
  int size{};
  std::cin >> size;

  std::string aabb{"aabb"};
  for (int i = 0; i < size; ++i) {
    std::cout << aabb[i % aabb.size()];
  }
  std::cout << "\n";

  return 0;
}